### PR TITLE
dash-bootstrap layout for image-segmentation app

### PR DESCRIPTION
**Pull request by surchs**
Issue for app: #501 

# App pull request
- [ ] This is a new app
- [X] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL : https://dash-playground.plotly.host/dash-image-segmentation/
- Current gallery app URL: https://dash-gallery.plotly.host/dash-image-segmentation/

## Workflow

- [X] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [X] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [X] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [X] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [X] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [X] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [ ] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-image-segmentation/app.py [hunk 1]
```diff
@@ -515,15 +640,27 @@ def annotation_react(
 
 # ======= Callback for modal popup =======
```
Comment (nicholas-esterer) [inline apps/dash-image-segmentation/app.py:641]:
```suggestion
# ======= Callback for modal popup =======
```
This is nitpicky (not a big deal haha) but maybe the comments could be normalized to all have the same style, e.g., all have === * === or not etc.

**Review by nicholas-esterer — APPROVED** [review]
This looks very good, much better looking than before! Just a small silly comment about comments, but otherwise :woman_dancing:

File: apps/dash-image-segmentation/app.py (hunk 1 above)
Comment (surchs) [reply apps/dash-image-segmentation/app.py:641]: I like that! Thanks for the suggestion! I think all other comments are just plain old `#`. So I'd say we take this one's fancy-style away. 
Did you change anything in your suggestion? I can't see the difference.

Comment (nicholas-esterer) [reply apps/dash-image-segmentation/app.py:641]: No just wanted to point out the comment style

**Comment by surchs** [comment]
ahem... @nicholas-esterer I managed to close the PR without merging. Could you maybe re-approve it?

**Comment by nicholas-esterer** [comment]
Sure! np

**Review by nicholas-esterer — APPROVED** [review]
:woman_dancing:

**Pull request merged**